app/models/session_format.rb

```diff
@@ -3,8 +3,7 @@ class SessionFormat < ActiveRecord::Base
   has_many :time_slots
   has_many :proposals
 
-  validates_presence_of :name, :event
-  validates_uniqueness_of :name, scope: :event
+  validates :name, uniqueness: {scope: :event}, presence: true
```

Comment:
I like the combining of presence and uniqueness validation here, though did you intend to drop the presence validation on :event?